numpy/doc/creation.py

```diff
@@ -104,7 +104,7 @@
 ::
 
  HDF5: PyTables
```

Comment:
Probably could update this with "h5py" instead of "PyTables"

Reply:
okay.